src/components/freelancerJobs/freelancerPage.tsx

```diff
@@ -72,12 +76,24 @@ const FreelancerPage: FC = () => {
 		));
 	}, [skillsOptions]);
 
+	if (isLoading) {
+		return <p>Loading...</p>;
+	}
+
+	const category = { value: userInfo.jobCategory.name, label: userInfo.jobCategory.name };
+	const skills = skillsMock.map(skill => ({
+		...skill,
+		value: userInfo.jobSkills.some((jobSkill: { name: string }) => jobSkill.name === skill.name),
+	}));
+	const filteredSkills = skillsOptions.filter(s => s.value);
+	const userSkills = filteredSkills.map(s => s.name);
```

Comment:
you are doing it with each rerender
its very very bad practice, change

Reply:
changed